extensions/2.0/Vendor/EXT_feature_metadata/README.md

```diff
@@ -246,7 +247,12 @@ A **feature** is a specific instantiation of class containing **property values*
 
 **Statistics** provide aggregate information about the metadata. For example, statistics may include the min/max values of a numeric property for mapping property values to color ramps or the number of enum occurrences for creating histograms.
 
-By default properties do not have any inherent meaning. A property may be assigned a **semantic**, an identifier that describes how the property should be interpreted. The full list of built-in semantics can be found in the [Cesium Metadata Semantic Reference](https://github.com/CesiumGS/3d-tiles/tree/3d-tiles-next/specification/Metadata/Semantics). Model authors may define their own application- or domain-specific semantics separately.
+By default properties do not have any inherent meaning. A property may be assigned a **semantic**, an identifier that describes how the property should be interpreted. Built-in semantics include `ID` and `NAME`, as defined below. Model authors may define their own application- or domain-specific semantics separately.
+
+- `ID`: Unique identifier for the feature.
+- `NAME`: Name of the feature; not required to be unique.
+
+> **TODO:** Is there any naming convention for application- or domain-specific semantics, such as an `_*` prefix?
```

Comment:
Not currently, though it's definitely something to consider to avoid name conflicts down the road. 

Another thing to consider is how semantics might be used. Certainly we want some built-in ones (ID, NAME). 
Down the road We might want to have other specs for official domain-specific things (e.g. maybe one group of semantics is for geospatial concepts like lat/lon/height or coordinate reference frame. Another group might be for attributes specific to weather data. Or LiDAR-related concepts. etc. etc.), whatever's broadly useful. those might want prefixes like `GEO_xxx` or `WEATHER_` or something.

But then vendors might want to have their own things. e.g. we would want some 3D Tiles specific things. And someday we'll probably have some metadata related to Cesium ion. And these will vary from what another vendor would want. But it would be nice to have a `CESIUM_ION_` prefix or something to group things like this together.

@sanjeetsuhag @lilleyse any further thoughts on this?

Reply:
Ok, we could just give some non-normative guidance for now, like:

 > "For application-specific semantics, authoring implementations are encouraged to use a `_*` prefix. For semantics common to a particular domain or vendor, creation of new uppercase, alphanumeric prefixes is encouraged."

Reply:
Added a (non-normative) note to the spec along these lines; I don't think it's necessary to define recommendations for prefixes in too much detail at this stage.